src/ApiMockerController.php

```diff
@@ -108,7 +108,13 @@ protected function getNewResponse()
 
         $status = array_get($this->config, 'status', 200);
 
-        return new Response($content, $status, ['Content-Type' => 'application/json']);
+        $header = ['Content-Type' => 'application/json'];
+
+        if (! empty($this->config['header'])) {
+            $header = $this->config['header'];
```

Comment:
```
$header = array_merge(
	$header,
	$this->config['header']
);
```

Reply:
could be multiple headers, so its fine by me...